StackoverflowChatbot/CommandProcessors/PriorityProcessor.cs

```diff
@@ -113,131 +75,21 @@ private bool TryGetNativeCommand(EventData data, out IAction? action)
 				return action != null;
 			}
 
+			// Why is action getting assigned but is unused?
 			action = null;
 			return false;
 		}
 
-		private BaseCommand CreateCommandInstance(Type commandType)
-		{
-			var parameterTypes = commandType
-				.GetConstructors()
-				.First()
-				.GetParameters()
-				.Select(e => e.ParameterType);
-
-			var parameterValues = new List<object>();
-			foreach(var param in parameterTypes)
-			{
-				if (param == typeof(ICommandStore))
-					parameterValues.Add(_commandStore);
-				else if (param == typeof(ICommandProcessor))
-					parameterValues.Add(this);
-			}
-
-			if (parameterValues.Any())
-				return (BaseCommand)Activator.CreateInstance(commandType, parameterValues.ToArray())!;
-
-			return (BaseCommand)Activator.CreateInstance(commandType)!;
-		}
-
-		private static bool IsCommand(string commandMessage, string commandKeyword, out string commandParameter)
-		{
-			var match = Regex.Match(commandMessage, $"^{commandKeyword} (.+)");
-
-			if (match.Success)
-			{
-				commandParameter = match.Groups[1].Value;
-				return true;
-			}
-
-			commandParameter = string.Empty;
-			return false;
-		}
-
-		private IAction LeaveRoomCommand(string commandParameter)
-		{
-			// Can be told to leave other rooms.
-			_roomService.LeaveRoom(IsSingleNumber(commandParameter.Trim(), out var room) ? room : _roomId);
-			return NewMessageAction(room > 0 ? $"Leaving room {room}!" : "Bye!");
-		}
-
-		// TODO wrap to Task
-		private IAction LearnCommand(string commandParameter)
-		{
-			var @params = commandParameter.Split(' ', 2, StringSplitOptions.RemoveEmptyEntries);
-			if (@params.Length < 2)
-			{
-				return NewMessageAction("Missing args");
-			}
-
-			var name = @params[0];
-			var args = @params[1];
-			var command = new CustomCommand(name, args);
-			if (DynamicCommand.TryParse(args, out var dynamicCommand))
-			{
-				command.IsDynamic = true;
-				command.ExpectedDynamicCommandArgs = dynamicCommand!.ExpectedArgsCount;
-			}
-
-			_ = _commandStore.AddCommand(command)
-				.ContinueWith(async t =>
```

Comment:
You'd miss that change from Learn.cs with a straightforward "resolve conflicts", because that whole code part is moved into Learn.cs, and diff doesn't search for conflicts between different files